Employee.py

```diff
@@ -48,31 +49,41 @@ def __init__(self, **kwargs) -> None:
             hourly (str, optional): _description_. Defaults to INVALID_STR.
             commission (str, optional): _description_. Defaults to INVALID_STR.
         """
-        self.name = kwargs.get("Name", INVALID_STR)
-        self.address = Address(**kwargs)
-        self.office_phone = kwargs.get("OfficePhone", INVALID_STR)
-        self.Emp_ID = kwargs.get("EmpID", INVALID_STR) if kwargs.get("ID", INVALID_STR) == INVALID_STR else kwargs.get("ID", INVALID_STR)
-        self.D_O_B = kwargs.get("DOB", INVALID_DATETIME)
-        self.SS_num = kwargs.get("SSN", -1)
-        self.Start_Date = kwargs.get("StartDate", INVALID_DATETIME)
-        self.End_Date = kwargs.get("EndDate", INVALID_DATETIME)
-        self.Permission_level = kwargs.get("Permission Level", 0)
-        self.Title = kwargs.get("Title", INVALID_STR)
-        self.Dept = kwargs.get("Dept", INVALID_STR)
-        self.Office_email = kwargs.get("Email", INVALID_STR)
-        self.active = kwargs.get("Archived", True)
-        self.permitted_lock_on = kwargs.get("Permitted", 0)
-        self.home_email = kwargs.get("HomeEmail", INVALID_STR)
-        self.home_phone = kwargs.get("HomePhone", INVALID_STR)
-        self.pay_method = kwargs.get("PayMethod", 1)
-        self.bank_info = kwargs.get("Account", INVALID_STR)
-        self.route = kwargs.get("Route", INVALID_STR)
-        self.salary = kwargs.get("Salary", INVALID_STR)
-        self.hourly = kwargs.get("Hourly", INVALID_STR)
-        self.commission = kwargs.get("Commission", INVALID_STR)
-        self.hashed_password =  kwargs.get("Password", INVALID_STR) if kwargs.get("Hashed Password", INVALID_STR) == INVALID_STR else kwargs.get("Hashed Password", INVALID_STR)
-        if self.hashed_password == INVALID_STR and self.Emp_ID != INVALID_STR:
```

Comment:
Why remove this line? I feel like the constructor should handle unset passwords, not any random spot of code that creates the employee.